src/workflow_renderer.jl

```diff
@@ -0,0 +1,205 @@
+
+# =============================================================================
+# Petri net viewer using GraphViz
+# =============================================================================
+"""
+    view_workflow(pnet::PetriNet, path::String)
+By default this method generates a png file after compiling the Petri net into an XML workflow and compiling the workflow.
```

Comment:
How is the output format selected by the user?

Reply:
I forgot to add that feature. Opps. Will work on it

Reply:
added options for more than one file format. But default remains PNG